javalib/src/main/scala/java/lang/Throwables.scala

```diff
@@ -10,15 +10,16 @@ private[lang] object StackTrace {
 
   private def makeStackTraceElement(
       cursor: Ptr[scala.Byte]): StackTraceElement = {
-    val name   = stackalloc[CChar](1024)
-    val offset = stackalloc[scala.Byte](8)
+    val nameMax = 1024
```

Comment:
extract to a consntant?

Reply:
It is used two places in the same method, none more generally. Original code had two literals. By training, I wanted to make sure both places used the same value.